src/main/java/io/github/cottonmc/energy/api/EnergyStorage.java

```diff
@@ -13,8 +13,14 @@
     int getCurrentEnergy();
 
     /**
-     * @return the highest tier of energy this device is capable of working with under any circumstances.
+     * @return the primary energy type to display in guis and probes.
      */
-    PacketTier getPacketTier();
+    EnergyType getPrimaryEnergyType();
+
+    /**
+     * @param type the type of energy that querying for connection.
+     * @return     whether that type of energy can interact with the storage.
+     */
+    boolean canConnectTo(EnergyType type);
```

Comment:
let's make this variadic so an especially industrious cable can ask, `canConnectTo(LV, MV, HV)`